packages/block-library/src/paragraph/edit.native.js

```diff
@@ -92,6 +92,7 @@ class ParagraphEdit extends Component {
 					tagName="p"
 					value={ content }
 					isSelected={ this.props.isSelected }
+					onFocus={ this.props.onFocus }
```

Comment:
I noticed as well that this has been included here in `Paragraph` block but [not in `Header` block](https://github.com/WordPress/gutenberg/blob/master/packages/block-library/src/heading/edit.native.js#L55). is there a reason not to add it to the `Header` block as well given that's the other RichText-based block currently in place? We should also remember to add this to all future RichText-based blocks that appear, so adding a comment might be good as well.
``` 
				<RichText
					tagName={ tagName }
					value={ content }
					isSelected={ this.props.isSelected }
					style={ {
						minHeight: Math.max( minHeight, this.state.aztecHeight ),
					} }
					onFocus={ this.props.onFocus } //always assign onFocus as a props
[...]
```

does that ☝️ make sense?